src/shared/ui/Button.tsx

```
@@ -0,0 +1,23 @@
+}
+
+export function Button({ title, isPending, pendingTitle = "확인중...", onClick, className }: ButtonProps) {
+  return (
+    <button
+      className={cn(
+        "cursor-pointer rounded-lg bg-blue-500 px-4 py-2 font-semibold text-white shadow-lg hover:bg-blue-400",
+        className
+      )}
+      onClick={onClick}
+    >
+      {isPending ? pendingTitle : title}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)